Source/WebCore/rendering/RenderLayerModelObject.cpp

```diff
@@ -287,6 +287,12 @@ TransformationMatrix* RenderLayerModelObject::layerTransform() const
 
 void RenderLayerModelObject::updateLayerTransform()
 {
+    if (is<RenderBox>(this) && style().offsetPath() && MotionPath::needsUpdateAfterContainingBlockLayout(*style().offsetPath())) {
+        if (auto cb = containingBlock()) {
```

Comment:
```suggestion
        if (auto containingBlock = this->containingBlock()) {
```

Let's avoid abbreviations per the webkit code style

Reply:
Fixed.